drivers/lora/Kconfig.sx12xx

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2019 Manivannan Sadhasivam
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+DT_COMPAT_SX1276 = semtech,sx1276
+DT_COMPAT_SX1261 = semtech,sx1261
+DT_COMPAT_SX1262 = semtech,sx1262
+
+menuconfig LORA_SX12XX
+	bool "Semtech SX-series driver"
+	select HAS_SEMTECH_RADIO_DRIVERS
+	select HAS_SEMTECH_LORAMAC
+	depends on SPI
+	help
+	  Enable LoRa driver for Semtech SX12xx.
+
+if LORA_SX12XX
+
+choice
+	prompt "LoRa Radio chipset"
+	default LORA_SX1276 if $(dt_compat_enabled,$(DT_COMPAT_SX1276))
+	default LORA_SX126X if $(dt_compat_enabled,$(DT_COMPAT_SX1261))
+	default LORA_SX126X if $(dt_compat_enabled,$(DT_COMPAT_SX1262))
+	help
+	  Antenna connection type.
+
+config LORA_SX1276
+	bool "Semtech SX1276 driver"
+	select HAS_SEMTECH_SX1276
+	help
+	  Enable LoRa driver for Semtech SX1276.
+
+config LORA_SX126X
+	bool "Semtech SX126x driver"
+	select HAS_SEMTECH_SX126X
+	help
+	  Enable LoRa driver for Semtech SX1261 and SX1262.
+
+endchoice
+
+choice
+	prompt "SX1276 PA Output pin"
```

Comment:
This looks like it might belong in the devicetree binding as a property, instead of in Kconfig, no?

Reply:
Simple answer -- yes. @mniestroj has created a pull request (#24169) to fix that. This pull request doesn't change the SX1276 driver other than factoring out common code to facilitate reuse.

Reply:
Great, thanks